gradle/libs.versions.toml

```diff
@@ -60,7 +60,7 @@ moshi-kotlinReflect = { module = "com.squareup.moshi:moshi-kotlin", version.ref
 okio = { module = "com.squareup.okio:okio", version = "3.8.0" }
 
 tools-common = { module = "com.android.tools:common", version.ref = "androidTools" }
-tools-layoutlib = { module = "com.android.tools.layoutlib:layoutlib-api", version = "31.2.2" }
+tools-layoutlib = { module = "com.android.tools.layoutlib:layoutlib-api", version = "31.3.0" }
```

Comment:
Closes https://github.com/cashapp/paparazzi/pull/1311

Reply:
Closes https://github.com/cashapp/paparazzi/pull/1348
If you pull in the patched version.

I think that might be relevant because they fixed compose preview related problems:
https://developer.android.com/studio/releases/fixed-bugs/studio/2023.2.1#android-studio-iguana-|-2023.2.1-patch-1

Reply:
* Fixes #1220

(note: comments and threads are not picked up automatically, so it's not enough me saying it here, you need to copy to OP or add to "Development" in sidebar.